versions/3.0.md

```diff
@@ -1646,38 +1646,32 @@ X-Rate-Limit-Reset:
     type: integer
 ```
 
-#### <a name="examplesObject"></a>Examples Object
+#### <a name="exampleObject"></a>Example Object
 
 Allows sharing examples for operation requests and responses.
```

Comment:
Since the fields table is dropped, consider explaining this is a free form object (possibly allowing primitive values as well?).